node/utils/constants.ts

```diff
@@ -0,0 +1,9 @@
+export const CUSTOMER_SCHEMA_NAME = 'CL'
+export const CUSTOMER_REQUIRED_FIELDS = [
+  'email',
+  'id',
+  'accountId',
+  'accountName',
+  'dataEntityId',
+]
```

Comment:
I'm hesitant about this, as it may be too restrictive. For example let's say a store decides to make the `firstName` field required. This would not be a problem for our app, since our user creation mutation already sets the `firstName` field. But because the field is not in this array, it would cause the warning message to be shown. 

Maybe the approach here should be `export const ALLOWED_CUSTOMER_REQUIRED_FIELDS`... like, list the fields that can be required without preventing our B2B apps from working.

Reply:
Ah! Good to know, this should work fine in that case.